### DISCUSSION TITLE
New Examples with 2.2.0 version for CSV and JSON

### DISCUSSION THREAD

**Discussion by kntpricing** [CMSgov/hospital-price-transparency#140] [Q&A] [answered]
Hello, could CMS please post examples for the Version 2.2.0 with all Jan. 1, 2025 requirements displayed for reference? Currently the only examples are for version 2.0.0 (Jan. 1, 2024 requirements). This examples are extremely helpful in assisting with fully comprehending how the data should be displayed. Thanks!

**Comment by RidenourK** [comment, marked as answer]
@kntpricing. Thank you for your question. 

Please see the current [examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples), which include examples of how to encode data for modifiers, drug unit and type of measurement, and the estimated allowed amount.
